models/aether_lat-lon/readme.rst

```diff
@@ -0,0 +1,191 @@
+The restart fields are divided among 2 types of files: neutrals and ions.
+They are further divided into "blocks", which are subdomains of the globe.
+Blocks start in the southwest corner of the lat/lon grid and go east first, 
+then to the west end of the next row north and end in the northeast corner. 
+Each block has a halo around it filled with field values from neighboring blocks.
+All of these need to be combined to make a single state vector for filter.
+There's a unique set of these files for each member.
+The restart file names reflect this information:  
+
+|   {neutrals,ions}_mMMMM_gBBBB.nc
+|   MMMM = ensemble member (0-based)
+|   BBBB = block number (0-based)
+
+The restart files do not have grid information in them, which must be read from
+   grid_gBBBB.nc
+
+Aether_to_dart and dart_to_aether read the same namelist; transform_state_nml.
+The fields chosen to be part of the model state are specified in ``variables``.
+Program aether_to_dart will read the specified fields, from all the restarts
+for a member plus grid files, and repackage them into an ensemble state vector file
+(filter_input.nc), which has a single domain and no halos.
+The field names will be transformed into CF-compliant names in filter_input.nc.
+
+Filter will read the ensemble of filter_input.nc files, assimilate, 
+and write an ensemble of filter_output.nc files.
+
+Dart_to_aether will convert the fields' names to the CF-compliant filter names,
+find those names in filter_output.nc, extract the updated field data, 
+and overwrite those fields in the appropriate Aether restart files.
+
+Namelists
+---------
+
+- The namelists are read from the file ``input.nml``. 
+- Namelists start with an ampersand '&' and terminate with a slash '/'.
+- Character strings that contain a '/' must be enclosed in quotes 
+  to prevent them from prematurely terminating the namelist.
+
+transform_state_nml
+...................
+
+   aether_restart_dirname 
+      The directory where the Aether restart files reside, 
+      and will be transformed (the "run" directory).
+
+   nblocks_lon, nblocks_lat, nblocks_lev 
+      Number of Aether domain "blocks" in the longitudinal, latitudinal, 
+      and vertical directions.  Vertical is always 1 (2024-2).
+      The total number of blocks (nblocks_lon x nblocks_lat x nblocks_lev)
+      is defined by the number of processors used by Aether.
+
+   variables
+      The Aether fields to be included in the model state are specified
+      in the ``variables`` namelist variable in transform_state_nml.
+      The following information must be provided for each field
+      
+         1) Aether field name
+         2) which file contains the field ("neutrals" or "ions")
+      
+      Aether field names are not CF-compliant and are translated 
+      to CF-compliant forms by aether_to_dart.
+      The suggested DART quantity to associate with some fields are listed
+      in ./aether_to_dart.nml.
```

Comment:
DART quality is not part of aether_to_dart or dart_to_aether, and the file aether_to_dart.nml does not exist.  
```suggestion
```